test/ngAnimate/animateCssDriverSpec.js

```diff
@@ -104,6 +104,14 @@ describe("ngAnimate $$animateCssDriver", function() {
         driver({ element: element });
         expect(capturedAnimation[1].applyClassesEarly).toBeFalsy();
       }));
+
+      it("should not provide a `method` as an option value if the animation is not structural", inject(function() {
```

Comment:
I agree with @Narretz here.  Let's change this to:

```js
it("should only set the event name if the animation is structural", inject(function() {
```

Also the previous it clause should probably change:

```js
      it("should signal $animateCss to apply the classes early when animation is structural", inject(function() {
```